src/DMCHandler.h

```diff
@@ -3,11 +3,52 @@
 
 #include <iostream>
 #include <primitives/block.h>
+#include "rpc/client.h"
+
+class DFITx {
+    private:
+    char* from; // TODO: get correct DMC Address type
+    char* to; // TODO: get correct DFI address type
+    int64_t amount;
+    char* signature;
+
+public:
+    DFITx(char* from, char* to, int64_t amount, char* signature){
```

Comment:
maybe check the types used in the native side RPCs to store hash values and use the same